test/controllers/concerns/lookup_keys_common_controller_test.rb

```diff
@@ -0,0 +1,36 @@
+require 'test_helper'
+
+class DummyController
+  cattr_accessor :callbacks
+  attr_accessor :params
+
+  def self.before_action(*args)
+    self.callbacks = args
+  end
+
+  include Api::V2::LookupKeysCommonController
+end
+
+class Api::V2::LookupKeysCommonControllerTest < ActiveSupport::TestCase
+  setup do
+    @dummy = DummyController.new
```

Comment:
You can use `tests DummyController` and have `@controller` set to it correctly, and then you can use `ActiveController::TestCase`